lib/oper.g

```diff
@@ -1758,8 +1782,24 @@ BIND_GLOBAL( "InstallGlobalFunction", function( arg )
         func := arg[2];
     fi;
     if IS_STRING( oper ) then
-      oper:= VALUE_GLOBAL( oper );
+      oper := VALUE_GLOBAL( oper );
     fi;
+
+ok := true;                                             ## for Utils 
+if ( oper in UTILS_FUNCTION_OPERS ) then                ## for Utils 
+    pos := 1;                                           ## for Utils 
+    while not ( oper = UTILS_FUNCTION_OPERS[pos] ) do   ## for Utils 
```

Comment:
Can't this be expressed more concisely as "pos := Position(UTILS_FUNCTION_OPERS, oper)? " 

Would also be nice to have some comments explaining what is being checked here. It's less than obvious just reading the code.